src/script/components/MessagesList/Message/index.tsx

```diff
@@ -88,6 +89,7 @@ export const Message: React.FC<MessageParams & {scrollTo?: ScrollToElement}> = p
     isHighlighted,
     hideHeader,
     onVisible,
+    onVisibilityChange,
```

Comment:
I know this is weird, but as `isVisible` was implemented till now, it would be triggered only the first time (and if it's added as item of the list one by one, and then disappears from the visible area - it will still be called). I don't mind to merge them at least on the `Message` and its usages level, but I want to hear your opinions on how to do it the best